pkg/subscriptions/subscriptions.html

```diff
@@ -0,0 +1,157 @@
+<html>
+<head>
+  <title>Subscriptions</title>
+  <meta charset="utf-8">
+  <link href="../base/cockpit.css" type="text/css" rel="stylesheet">
+  <script src="../base/cockpit.js"></script>
+</head>
+<body>
+    <!-- different containers, depending on current subscription status: updating, unregistered, registered -->
+    <div id="subscriptions-register-failure" class="container-fluid" hidden>
+      <div class="alert alert-danger" role="alert">
+        <span class="pficon-layered > pficon pficon-error-octagon + pficon pficon-error-exclamation"></span>
+          <span><b>System couldn't be registered.</b></span>
+          <span id="subscriptions-register-failure-reason"></span>
+        <button type="button" class="close" aria-label="Close"
+                onclick="$('#subscriptions-register-failure').hide();">
+          <span class="pficon pficon-close"/>
+        </button>
+      </div>
+    </div>
+    <div id="subscriptions-status-subscribe-success" class="container-fluid" hidden>
+      <div class="alert alert-success" role="alert">
+        <span class="pficon pficon-ok"></span>
+        <span><b>System successfully subscribed</b> You will now receive software updates.</span>
+        <button type="button" class="close" aria-label="Close"
+                onclick="$('#subscriptions-status-subscribe-success').hide();">
+          <span class="pficon pficon-close"></span>
+        </button>
+      </div>
+    </div>
+    <div id="subscriptions-status-subscribe-failure" class="container-fluid" hidden>
+      <div class="alert alert-danger" role="alert">
+        <span class="pficon-layered > pficon pficon-error-octagon + pficon pficon-error-exclamation"></span>
+          <span><b>System isn't subscribed.</b></span>
+          <span id="subscriptions-status-subscribe-failure-reason">You cannot receive software updates.</span>
+        <button type="button" class="close" aria-label="Close"
+                onclick="$('#subscriptions-status-subscribe-failure').hide();">
+          <span class="pficon pficon-close"/>
+        </button>
+      </div>
+    </div>
+    <div id="subscriptions-unregistered" class="container-fluid" hidden>
+      <span style="display: block; text-align: center;">
```

Comment:
Suggest adding a semantic class and a definition in a ```<style>``` block above instead of an inline style.